packages/@aws-cdk/aws-stepfunctions-tasks/test/codebuild/batch-delete-builds.test.ts

```diff
@@ -0,0 +1,62 @@
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Stack } from '@aws-cdk/core';
+import { CodeBuildBatchDeleteBuilds } from '../../lib/codebuild/batch-delete-builds';
+
+let stack: Stack;
+let buildIds: string[];
+
+beforeEach(() => {
+  //GIVEN
+  stack = new Stack();
+  buildIds = ['buildId1', 'buildId2'];
```

Comment:
add a unit test that uses a `path` (I did see it in the integ test, but we should also have a unit test for it)